### PR TITLE
Full update to project --> associated projects

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Updated to remove old angular code and to replace with newer ant design look at feel.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/webapp/resources/js/apis/projects/projects.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-      `${window.TL.BASE_URL}ajax/projects?admin=${window.location.href.includes(
-        "all"
-      )}
+      `${URL}?admin=${window.location.href.includes("all")}
 `,
       params
     )
     .then(response => response.data);
+
+export const getAssociatedProjects = id =>
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/apis/projects/projects.js]: Comments for functions

File: src/main/webapp/resources/js/apis/projects/projects.js [hunk 2]
```diff
@@ -1,12 +1,25 @@
 import axios from "axios";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/apis/projects/projects.js]: File comment

File: src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx]: File comment

File: src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx]: File comment

File: src/main/webapp/resources/js/components/Tables/fitlers/TextFilter.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Tables/fitlers/TextFilter.js]: File comment

File: src/main/webapp/resources/js/components/Tables/filter-utilities.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import uniqBy from "lodash/uniqBy";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Tables/filter-utilities.js]: File comment

File: src/main/webapp/resources/js/components/Buttons/ConfirmButton.jsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Buttons/ConfirmButton.jsx]: File comment and when to use this

File: src/main/webapp/resources/js/utilities/link-utilities.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/utilities/link-utilities.js]: File comment

File: src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx (hunk 3 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx]: Updated in 6262caf9c9e3bddc898cb212dca62e4f0b1fa543

File: src/main/webapp/resources/js/apis/projects/projects.js (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/apis/projects/projects.js]: Updated in 3c6628b941237996c513d700a6624d91b58e23c9

File: src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx (hunk 3 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx]: Updated in c7edce73cbc8f57b149a0692c5b2d738fcbda5d5

File: src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx (hunk 4 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx]: Updated in c7edce73cbc8f57b149a0692c5b2d738fcbda5d5

File: src/main/webapp/resources/js/components/Tables/fitlers/TextFilter.js (hunk 5 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Tables/fitlers/TextFilter.js]: Updated in 3ce9d15384edebcbbc46221a282033890d0ec604

File: src/main/webapp/resources/js/components/Tables/filter-utilities.js (hunk 6 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Tables/filter-utilities.js]: Updated in d5afbb8fb56a28350db36d77116c7a41f5c993dd

File: src/main/webapp/resources/js/components/Buttons/ConfirmButton.jsx (hunk 7 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Buttons/ConfirmButton.jsx]: Updated in bbe476aac3c33275094afab6d2be49b9fd570d15

File: src/main/webapp/resources/js/utilities/link-utilities.js (hunk 8 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/utilities/link-utilities.js]: Updated in ae0c96c5abfdd9cd32d193785314afd51f544a1e

File: src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/projects/ProjectSettingsAjaxControllerTest.java [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+		associatedProject.setId(2L);
+
+		when(projectService.read(1L)).thenReturn(project);
+		when(projectService.read(2L)).thenReturn(associatedProject);
+		controller.removeAssociatedProject(1L, 2L);
+		verify(projectService, times(1)).removeRelatedProject(project, associatedProject);
+	}
+
+	@Test
+	public void testAssociateProject() {
```
Comment (deepsidhu85) [inline src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/projects/ProjectSettingsAjaxControllerTest.java]: Change to testAssociatedProject

**Pull request merged**